l10n_be_mis_reports/data/mis_report_vat.xml

```diff
@@ -0,0 +1,369 @@
+      <field name="sequence">30</field>
+    </record>
+    <record model="mis.report.kpi" id="mis_report_vat_grid_03">
+      <field name="report_id" ref="mis_report_vat"/>
+      <field name="name">grid_03</field>
+      <field name="description">Grid 03</field>
+      <field name="expression">-bal[][('tax_ids.tag_ids.name', 'like', '% 03'),('credit','>',0)]</field>
+      <field name="type">num</field>
+      <field name="compare_method">pct</field>
+      <field name="style_id" ref="mis_report_style_l10n_be_3"/>
+      <field name="sequence">40</field>
+    </record>
+    <record model="mis.report.kpi" id="mis_report_vat_grid_44">
+      <field name="report_id" ref="mis_report_vat"/>
+      <field name="name">grid_44</field>
+      <field name="description">Grid 44</field>
+      <field name="expression">-bal[][('tax_ids.tag_ids.name', 'like', '% 44'),('credit','>',0)] </field>
+      <field name="type">num</field>
+      <field name="compare_method">pct</field>
+      <field name="style_id" ref="mis_report_style_l10n_be_3"/>
+      <field name="sequence">50</field>
+    </record>
+    <record model="mis.report.kpi" id="mis_report_vat_grid_45">
+      <field name="report_id" ref="mis_report_vat"/>
+      <field name="name">grid_45</field>
+      <field name="description">Grid 45</field>
+      <field name="expression">-bal[][('tax_ids.tag_ids.name', 'like', '% 45'),('credit','>',0)]</field>
+      <field name="type">num</field>
+      <field name="compare_method">pct</field>
+      <field name="style_id" ref="mis_report_style_l10n_be_3"/>
+      <field name="sequence">60</field>
+    </record>
+    <record model="mis.report.kpi" id="mis_report_vat_grid_46">
+      <field name="report_id" ref="mis_report_vat"/>
+      <field name="name">grid_46</field>
+      <field name="description">Grid 46</field>
+      <field name="expression">-bal[][('tax_ids.tag_ids.name', 'like', '% 46'),('credit','>',0)]</field>
+      <field name="type">num</field>
+      <field name="compare_method">pct</field>
+      <field name="style_id" ref="mis_report_style_l10n_be_3"/>
+      <field name="sequence">70</field>
+    </record>
+    <record model="mis.report.kpi" id="mis_report_vat_grid_47">
+      <field name="report_id" ref="mis_report_vat"/>
+      <field name="name">grid_47</field>
+      <field name="description">Grid 47</field>
+      <field name="expression">-bal[][('tax_ids.tag_ids.name', 'like', '% 47'),('credit','>',0)]</field>
+      <field name="type">num</field>
+      <field name="compare_method">pct</field>
+      <field name="style_id" ref="mis_report_style_l10n_be_3"/>
+      <field name="sequence">80</field>
+    </record>
+    <record model="mis.report.kpi" id="mis_report_vat_grid_48">
+      <field name="report_id" ref="mis_report_vat"/>
+      <field name="name">grid_48</field>
+      <field name="description">Grid 48</field>
+      <field name="expression">bal[][('tax_ids.tag_ids.name', 'like', '% 48'),('debit','>',0)] </field>
```

Comment:
For cases like this where we want only debits, it's better to use `deb[][('tax_ids.tag_ids.name', 'like', '% 48')]`. Performance will be better.